tests/network-tests/src/scenarios/full.ts

```diff
@@ -47,9 +47,7 @@ scenario('Full', async ({ job, env }) => {
     ? job('runtime upgrade proposal', runtimeUpgradeProposal).requires(councilJob)
     : undefined
 
-  const coreJob = runtimeUpgradeProposalJob || councilJob
-
-  // All other jobs should be executed after coreJob
+  const coreJob = councilJob || runtimeUpgradeProposalJob
```

Comment:
This change is not correct.
If a runtime upgrade job is running then the core job is the runtime upgrade job (which itself depends on council job). Or you will have tests potentially running before and after upgrade.